### PR TITLE
Project and About me Section

### PR DESCRIPTION
**Pull request by Newtayo**
Portfolio Project Milestone 2
The following features have been added to portfolio website.

- Portfolio website now has the details of each projects.
- The about me page was added to provide a brief description of my experience
- The languages, frameworks and skills have also been added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
     </section>
     
 </div>
+    <div class ="worksection">
+        <section id = "worksectionheader">
+        <h1 id= "worksectionheading"> My Recent Works</h1>
```
Comment (bobb-Rob) [inline index.html]: - [ ] The title (e.g., "My Recent Works") uses a header tag hx. You could use `h2` tag. This is essentially due to SEO reasons which many believe it is better to have only one `h1 tag on a webpage.

File: index.html [hunk 2]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+        <img id="sectiondivider" src ="image/Vector (10).png">
+        </section>
+        
+        <section class="workdescription" >
```
Comment (bobb-Rob) [inline index.html]: - [ ] Your project card has red borders on them. I believe you used them during styling and then forgot to remove them afterward. Kindly remove them.

File: index.html [hunk 3]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+                <h2 id="project1text">Multi-Post Stories</h2>
+                <p id="project1supporttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul id ="project1language">
+                    <li class ="languageitem">css</li>
+                    <li class ="languageitem">html</li>
+                    <li class ="languageitem" id ="bootstrap">bootstrap</li>
+                    <li class ="languageitem">Ruby</li>
+                </ul>
+                <button id="seeproject1">See Project</button>
+            </div>
```
Comment (bobb-Rob) [inline index.html]: - [ ] Your design is different from the provided Figma design in many ways. Please check for all margins and heights. I see you use a lot of fixed heights (view height). You should not do that for many reasons, but most importantly, allow the height of your element to grow naturally with its contents unless you truly need a fixed height. In the case of this project cards. you should allow your cards to have responsive height.

Figma Design | Your Design
---- | ----
image | image

File: index.html [hunk 4]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
```
Comment (bobb-Rob) [inline index.html]: - [ ] Your HTML Structure is not using the right HTML semantics. The section element is a generic standalone section of the document, so use it to group elements that are related together. To know more about section tag [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
Please see below the simple structure that I suggest you follow for the works section of the portfolio: 

```
<section class="works">
    <div>
      <h2>My recent works</h2>
      <hr>
    </div>
    <!-- align the projects cards of the div below with display:grid -->
    <div class="grid here"> 
      <div class="project-cards feature-card">
        <img src="" alt="">
        <h2>Multipost Stories</h2>
        <p>Lorem ipsum dolor sit amet consectetur.</p>
        <ul>
          <li>CSS</li>
          <li>HTML</li>
          <li>Bootstrap</li>
          <li>Ruby</li>
        </ul>
        <button type="button">See project</button>
      </div>
      
      <div class="project-cards"></div>
      <div class="project-cards"></div>
      <div class="project-cards"></div>
      <div class="project-cards"></div>
      <div class="project-cards"></div>
    </div>
  </section>
```

File: index.html [hunk 5]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+            </ul>
+            <button class="seeproject2">See Project</button>
+        </div>
+    </section>
+        
+    </div>
+    <div class="aboutsection" >
+        <section class ="about">
 
+            <h1 class="aboutheader"> About me</h1>
```
Comment (bobb-Rob) [inline index.html]: - [ ] Please refer to my previous comment on multiple `h1` tags on a webpage. Kindly replace this `h1` with another header tag, maybe `h2`.

File: index.html [hunk 6]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+            <p id ="aboutmetext">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button id="getresumebutton">Get My Resume</button>
```
Comment (bobb-Rob) [inline index.html]: - [ ] It is best practice to ensure every HTML button has type property. Kindly update this button tag with a `type="button"`. Please ensure the button looks exactly like what you have on Figma design. Please remove border.

**Figma**
image

Your Button 
image

File: index.html [hunk 7]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+                        <li class="list">Github</li>
+                        <li class="list">Codepen</li>
+                        <li class="list">Gitlab</li>
+                        <li class="list">Terminal</li>
+                    </ul>
+                    
+                </li>
+            </ul>
+
+        </section>
```
Comment (bobb-Rob) [inline index.html]: 👏  This is an excellent nesting you have here. Well done.

File: index.html [hunk 8]
```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
...
+                <li class ="language2item">html</li>
+                <li class ="language2item" id ="bootstrap">bootstrap</li>
+                <li class ="language2item">Ruby</li>
+            </ul>
+            <button class="seeproject2">See Project</button>
+        </div>
+    </section>
+        
+    </div>
+    <div class="aboutsection" >
```
Comment (bobb-Rob) [inline index.html]: - [ ] Section element should be the top-level wrapper for the about section and not a div. This is pure HTML semantic use of elements.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Newtayo , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Newtayo** [comment]
- Semantic html applied to work and about me section
- buttons type has been set to button
- image has been made responsive

**Review by IngDavidTM — APPROVED** [review]
Hi @Newtayo  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions


- [OPTIONAL] You did a wonderful job with all the requirements suggested :+1: :100:, but I have noticed that the height of each card is not matching with the Figma template.

**Your design**
Screenshot from 2023-01-14 12-04-21
**Figma design**
Screenshot from 2023-01-14 12-04-46


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**